lib/Models/SensorObservationServiceCatalogItem.ts

```diff
@@ -624,7 +624,7 @@ function createChartColumn(
   name: string | undefined
 ): string {
   const nameAttr = name == undefined ? "" : `name="${name}"`;
-  return `<sos-chart identifier="${identifier}" ${nameAttr}></sos-chart>`;
+  return `<sos-chart identifier="${identifier}" ${nameAttr} can-download="false"></sos-chart>`;
```

Comment:
Can you explain this change?

Reply:
The API that provides the chart data is a SOAP API, and the download button is essentially just a link, so when you click it you get an error page. I asked Nanda if it ever worked for SOS, and he said no. So essentially, this change hides a button that doesn't work, and hasn't ever worked.
This doesn't stop you from downloading the chart once you've expanded it, though.

Reply:
Ok. Sounds like this change is an improvement on the current state then. Ideally sos-chart would default to no download button, or provide the ability to download data without expanding, but I don't think we need to do that now.